[DOCS] New grouped navigation side menu (#2088)
elastic/security-docs: docs/getting-started/advanced-setting.asciidoc

The advanced settings determine:

* Which indices {es-sec} uses to retrieve data
* {ml-cap} anomaly score display threshold

> * The navigation UI used throughout the {security-app}

Author: ```suggestion
* The navigation menu style used throughout the {security-app}
```

To learn more, refer to our https://www.elastic.co/legal/privacy-statement[Privacy Statement].

[discrete]
== Set machine learning score threshold

When security <<machine-learning, {ml} jobs>> are enabled, this setting

> determines the threshold above which anomaly scores are displayed in {es-sec}:

Author: @benironside In my previous job, our style guide banned "appear" in favor of "is/are displayed," and it's been a hard habit for me to break!

* `securitySolution:defaultAnomalyScore`

[discrete]
== Enable grouped navigation

> The `securitySolution:enableGroupedNav` setting enables a new navigation UI for {elastic-sec} that groups related pages and highlights commonly visited areas for a streamlined experience.

Author: ```suggestion
The `securitySolution:enableGroupedNav` setting enables the grouped navigation menu for {elastic-sec}, which groups related pages and highlights commonly visited areas for a streamlined experience.
```

> The `securitySolution:enableGroupedNav` setting enables a new navigation UI for {elastic-sec} that groups related pages and highlights commonly visited areas for a streamlined experience.
> 
> By default, the {security-app} uses the legacy navigation UI; turn on `securitySolution:enableGroupedNav` to use the grouped navigation UI.

Author: ```suggestion
The `securitySolution:enableGroupedNav` setting enables the grouped navigation menu for {elastic-sec}, which groups related pages and highlights commonly visited areas for a streamlined experience.

By default, the {security-app} uses the legacy navigation menu; turn on `securitySolution:enableGroupedNav` to use the grouped navigation menu.
```
